front/app/api/authentication/confirm_email/resendEmailConfirmationCode.ts

```diff
@@ -1,7 +1,15 @@
 import { API_PATH } from 'containers/App/constants';
+import fetcher from 'utils/cl-react-query/fetcher';
 import streams from 'utils/streams';
```

Comment:
In this file there is one other place where streams are used (`streams.fetchAllWith`). I don't think we can replace this with anything yet.